n2v/inverter.py

```diff
@@ -384,7 +387,25 @@ def generate_components(self, guide_potential_components):
         self.vb = np.zeros_like(self.T)
 
         N = self.nalpha + self.nbeta
-        self.J0, _ = self.form_jk(self.ct[0], self.ct[1])
+
+        if self.J0 is None:
```

Comment:
I know you mentioned the use for natural orbitals, but can you direct me to some references that you used to understand it better?

Reply:
I don't have a paper for this and it is strictly not natural orbitals. The main idea is to find effective orbitals and effective (fractional) occupations that will reproduce the correct density matrices.